hosting/src/events/Events/EventRow.html

```diff
@@ -0,0 +1,64 @@
+<div class="py-4 px-4 ring-2 ring-gray-100 rounded-xl mb-6 bg-gray-50">
+  <div class="grid grid-cols-1 sm:grid-cols-3 gap-x-5 gap-y-4">
+    <div class="col-span-1">
+      <div class="aspect-w-16 aspect-h-9 w-full">
+        <div
+          class="w-full h-full bg-cover bg-top rounded-lg shadow-lg"
+          :style="event.bannerUrl 
+              ? { backgroundImage: 'url(' + event.bannerUrl + ')' } 
+              : { backgroundImage: 'url(/images/banner_16_9_default.jpg)'}"
+        ></div>
+      </div>
+      <p class="mt-3 text-center">
+        <router-link
+          v-if="true"
+          :to="{ name: 'Event', params: { slug: event.slug } }"
+          class="text-brand-blue-900 hover:text-brand-blue-500"
+          >&gt; &gt; Register Now &lt; &lt;</router-link
+        >
+      </p>
```

Comment:
This is the most important thing to hide right now

Reply:
Done